azurerm/internal/services/mariadb/mariadb_server_resource_test.go

```diff
@@ -237,7 +237,6 @@ func TestAccMariaDbServer_createReplica(t *testing.T) {
 			Config: r.createReplica(data, version),
 			Check: resource.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
-				check.That("azurerm_mariadb_server").ExistsInAzure(r),
```

Comment:
Should we instead check that the replica exists? `check.That("azurerm_mariadb_server.replica").ExistsInAzure(r),`

Reply:
Ooo yes. Good call